src/common/components/HeroSection/HeroSection.tsx

```diff
@@ -0,0 +1,12 @@
+import Link from "next/link";
```

Comment:
This component is not included in PR description. I think this component can be added in a separate PR.

Reply:
Thanks resolve it